runtime/compiler/build/toolcfg/gnu/common.mk

```diff
@@ -226,7 +226,7 @@ S_DEFINES+=$(HOST_DEFINES) $(TARGET_DEFINES)
 S_DEFINES_DEBUG+=DEBUG
 
 S_FLAGS+=--noexecstack
-S_FLAGS_DEBUG+=--gstabs
+S_FLAGS_DEBUG+=--gstabs+ -g
```

Comment:
Do we need to update CMake as well? @dnakamura FYI. Also will this flag affect performance? Not sure if gcc would do any performance passes on an assembly file. My guess would be no but worth asking.

Reply:
> Also will this flag affect performance? Not sure if gcc would do any performance passes on an assembly file. 

I do not think so, it would and even this would added to debug build when we build the JIT with `BUILD_CONFIG=debug` and won't affect the prod builds. 
The only concern  I have is the statement on the man page which says it will break other live debuggers on Linux, I am not aware of any other we use.